File: Deal/src/main/java/ru/leonov/deal/model/record/AdditionalServicesRecord.java
package ru.leonov.deal.model.record;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * POJO that contain information about additional services applied to client.
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class AdditionalServicesRecord {

    /**
     * Did client purchase insurance?
     */
    private Boolean isInsuranceEnabled;

    /**
     * Is this client salary client?
     */
    private Boolean isSalaryClient;
}

File: Deal/src/main/kotlin/ru/leonov/deal/model/record/AdditionalServicesRecord.kt
package ru.leonov.deal.model.record

/**
 * POJO that contain information about additional services applied to client.
 */
data class AdditionalServicesRecord(
    /**
     * Did client purchase insurance?
     */
    val isInsuranceEnabled: Boolean,
    /**
     * Is this client salary client?
     */
    val isSalaryClient: Boolean
)
